packages/http/src/validator/validators/__tests__/__snapshots__/utils.spec.ts.snap

```diff
@@ -6,6 +6,7 @@ Array [
     "code": "required",
     "message": "c is required",
     "path": Array [
+      "a",
```

Comment:
@XVincentX This update to the snapshot is (similarly to #1530) a side effect of the proposed change.

That new path segment looks legit to me as the unit test leveraging it is:

```ts
  const errorObjectFixture: ErrorObject = {
    dataPath: 'a.b',
    keyword: 'required',
    message: 'c is required',
    schemaPath: '..',
    params: {},
  };

  describe('all fields defined', () => {
    it('converts properly', () => {
      expect(convertAjvErrors([errorObjectFixture], DiagnosticSeverity.Error)).toMatchSnapshot();
    });
  });
```